test/references/CKAN Consumer Complaint.txt

```diff
@@ -1,12 +1,12 @@
 register(DataDep(
     "Consumer Complaint Database",
     """
-    
-Website: https://catalog.data.gov/dataset/consumer-complaint-database
-Author: 	
-Maintainer: 	devops@cfpb.gov
-License: 	License not specified
-Date of Creation: 	February 26, 2014
+Website:	https://catalog.data.gov/dataset/consumer-complaint-database
+Author:	
```

Comment:
Looks like something is going wrong with the Author here.
I am guessing it is an empty, string
we might need a dispatch that catches that, and re-dispatches it as a missing